### PR TITLE
docs(proxy-cache-advanced): add ignore_uri_case

### PR DESCRIPTION
**Pull request by samugi**
### Description

Add `ignore_uri_case` configuration parameter to `proxy-cache-advanced`, similar to https://github.com/Kong/docs.konghq.com/pull/5270/files (for proxy-cache).

### Testing instructions

Netlify link: 

### Checklist 

- [X] Review label added
- [X] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/_hub/kong-inc/proxy-cache-advanced/_index.md [hunk 1]
```diff
@@ -86,6 +86,13 @@ params:
       datatype: boolean
       description: |
         When enabled, respect the Cache-Control behaviors defined in [RFC7234](https://tools.ietf.org/html/rfc7234#section-5.2).
+    - name: ignore_uri_case
+      required: false
+      default: false
+      value_in_examples: null
+      datatype: boolean
+      description: |
+        If set to `true`, requests will be cached while ignoring case sensitivity in the URI.
```
Comment (lena-larionova) [inline app/_hub/kong-inc/proxy-cache-advanced/_index.md:95]:
```suggestion
        Determines whether to treat URIs as case sensitive. By default, case sensitivity is enabled. If set to `true`, requests are cached while ignoring case sensitivity in the URI.
```
How about something like this? The initial description feels like it's missing some context.

**Review by lena-larionova — CHANGES_REQUESTED** [review]
Can you please add a changelog entry to the bottom of the same file?

**Comment by samugi** [comment]
all done @lena-larionova, thank you!

**Review by lena-larionova — APPROVED** [review]
LGTM!

File: app/_hub/kong-inc/proxy-cache-advanced/_index.md [hunk 2]
```diff
@@ -86,6 +86,13 @@ params:
       datatype: boolean
       description: |
         When enabled, respect the Cache-Control behaviors defined in [RFC7234](https://tools.ietf.org/html/rfc7234#section-5.2).
+    - name: ignore_uri_case
```
Comment (lena-larionova) [inline app/_hub/kong-inc/proxy-cache-advanced/_index.md:89]:
```suggestion
    - name: ignore_uri_case
      minimum_version: "3.3.x"
```
Missed one thing.

**Pull request merged**